packages/server/src/app.ts

```diff
@@ -18,7 +18,9 @@ const { logAlert } = require("@budibase/backend-core/logging")
 const { Thread } = require("./threads")
 import redis from "./utilities/redis"
 import * as migrations from "./migrations"
-import { events, installation } from "@budibase/backend-core"
+import { events, installation, tenancy } from "@budibase/backend-core"
+import { createAdminUser, getChecklist } from "./utilities/workerRequests"
+import { tenantSucceeded } from "@budibase/backend-core/dist/src/events/publishers/backfill"
```

Comment:
This might cause problems in local dev, we shouldn't have a `dist` import in here. This looks unused so could be removed

Reply:
> What about adding this logic to worker instead? Where worker is the owner of the user domain currently. It would prevent introducing additional server -> worker api requests

Yea I originally had it in the worker, but the problem with that is that it depends on the server being up. The worker is usually ready first so it was failing when trying to run this new stuff. Can't remember exactly which API endpoint it was trying to call but yea it depends on the server being up. I didn't really like the idea of just waiting around and pinging the server on an interval to check if it was up, so I figured it would be cleaner to instead put this in the server.